deep_quoridor/src/agents/alphazero/nn_evaluator.py

```diff
@@ -77,10 +87,12 @@ def evaluate_tensors(
 
     @torch.no_grad
     def evaluate_batch(self, games: list[Quoridor]):
+        start_time = time.time()
         hashes = [g.get_fast_hash() for g in games]
 
         if all([h in self.cache for h in hashes]):
             # everything was cached!
+            self.evaluation_infos.append(EvaluationInfo(start_time=start_time, cache_hit=True))
```

Comment:
I don't set the end_time for cache hits, since I assume these are very fast and we only want to measure the durations of actual GPU evaluations.